apps/vc-api/src/config/env-vars-validation-schema.ts

```diff
@@ -17,8 +17,40 @@
 
 import * as Joi from 'joi';
 
+export enum DB_TYPES {
+  SQLITE_INMEMORY = 'SQLITE_INMEMORY',
```

Comment:
Perhaps a bit more consistent would be to  use same values as in `TypeOrmModuleOptions.type`

Reply:
Hmm... the `TypeOrmModuleOptions.type` does not have anything for in-memory SQLite. So, maybe having this completely different is less confusing? What do you think?

Reply:
Can we use this https://github.com/typeorm/typeorm/blob/master/src/driver/types/DatabaseType.ts?

Reply:
We could if we had something there for both `SQLITE_IN_MEMORY` and `SQLITE`. I see there is only a `sqlite` value there.